sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -14,26 +14,19 @@ namespace Azure.DigitalTwins.Core.Samples
 {
     internal class ComponentSamples
     {
-        private DigitalTwinsClient DigitalTwinsClient { get; }
-
-        public ComponentSamples(DigitalTwinsClient dtClient)
-        {
-            DigitalTwinsClient = dtClient;
-        }
-
         /// <summary>
         /// Creates a digital twin with Component and upates Component
         /// </summary>
-        public async Task RunSamplesAsync()
+        public async Task RunSamplesAsync(DigitalTwinsClient client)
         {
             PrintHeader("COMPONENT SAMPLE");
 
             // For the purpose of this example we will create temporary models using a random model Ids.
             // We have to make sure these model Ids are unique within the DT instance.
 
-            string componentModelId = await GetUniqueModelIdAsync(SamplesConstants.TemporaryComponentModelPrefix, DigitalTwinsClient).ConfigureAwait(false);
```

Comment:
Azure SDK team said leave off ConfigureAwait(false) on sample code.

But leave it in library code.